coalib/output/ConsoleInteraction.py

```diff
@@ -749,6 +749,8 @@ def show_bear(bear,
                          '  ',
                          'This bear cannot fix issues or does not provide '
                          'information about what categories it can fix.')
+        console_printer.print(
+            '  Path:\n' + '   ' + bear.file_path + '\n\n')
```

Comment:
Quotes around the file path? Spaces are possible. Or better - quotes *if* spaces exist, no otherwise. Don't change the `file_path` property function though - we might need the raw filename later on.